src/libraries/Common/tests/Tests/System/Net/aspnetcore/Http2/HuffmanDecodingTests.cs

```diff
@@ -354,29 +356,6 @@ public void HuffmanEncode(int code, uint expectedEncoded, int expectedBitLength)
             Assert.Equal(expectedBitLength, bitLength);
         }
 
-        [Theory]
-        [MemberData(nameof(HuffmanData))]
```

Comment:
Why were these removed?

Reply:
Method Huffman.DecodeValue has been deleted during rewriting decode. It was, if I understand it correctly, internal only for purpose of unit testing this particular method and its contract was not used anywhere else, apart from previous Decode and these tests.

When I was deleting it, I was thinking of rewriting those test to use Huffman.Decode method however I realized that all test cases that this deleted tests have covered are also covered in existing HuffmanDecoding_ValidEncoding_Succeeds test which test cases are including all single element array of chars with value 0..255.

Reply:
Ok, as long as we have coverage for all the cases that are removed, it's fine to get rid of these test cases.